cmd/openshift-install/create.go

```diff
@@ -313,14 +314,15 @@ func waitForConsole(ctx context.Context, config *rest.Config, directory string)
 			if silenceRemaining == 0 {
 				logrus.Debug("Still waiting for the console route...")
 				silenceRemaining = logDownsample
+				err = errors.New("no routes found in openshift-console namespace")
 			}
 		}
 	}, 2*time.Second, consoleRouteContext.Done())
 	if err != nil {
-		return "", errors.Wrap(err, "waiting for console route to be created")
+		return "", err
```

Comment:
Can you motivate this change?  Do you know what the error looked like before?

Reply:
I've never seen the installer get to that msg.  With the bug in question in this PR, I only saw this:
```console
DEBUG Destroy complete! Resources: 9 destroyed.    
INFO Waiting 1s for the openshift-console route to be created... 
DEBUG Still waiting for the console route...       
DEBUG Still waiting for the console route...       
DEBUG Still waiting for the console route...       
DEBUG Still waiting for the console route...       
INFO Install complete!                            
INFO Run 'export KUBECONFIG=/home/somalley/installer/somalley/auth/kubeconfig' to manage the cluster with 'oc', the OpenShift CLI. 
INFO The cluster is ready when 'oc login -u kubeadmin -p 6kcEf-bZsdv-o6gct-4qUm5' succeeds (wait a few minutes). 
INFO Access the OpenShift web-console here:       
INFO Login to the console with user: kubeadmin, password: 6kcEf-bZsdv-o6gct-4qUm5 
```
(I've been reproducing by running from master but changing the timeout to 1 sec, retries every millisecond)